share/completions/figlet.fish

```diff
@@ -0,0 +1,29 @@
+function __fish_print_figlet_fonts
+    find (figlet -I2) -type f ^ /dev/null | sed -e 's$/.*/\([^/.]*\)\.[ft]lf$\1\tFont$;tx;d;:x'
```

Comment:
What does `figlet -I2` output?

This might be nicer with string, though it's something we can also do later.